### PR TITLE
feature: add api documentation capability

### PR DESCRIPTION
**Pull request by izumizawa**
I added how to create the database in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
-# Cadastro-central
+# Central register
+
```
Comment (gabrieloswaldo) [inline README.md:2]: Maybe add "About" section here, describing **succinctly** what is the central_register, what it does, etc.

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+#### Server
+* Host: `localhost`
+* Port: `5432`
+* Database name: `central_register`
+#### Authentication
+* Authentication: `Database Native`
+* Username: `postgres`
+* Password: `postgres`
+
+With the created base, you just need to run the application, and it will create the tables by itself because we are using Liquibase library. After that you can access our documentation as shown below.
```
Comment (gabrieloswaldo) [inline README.md]: Maybe just add here the reference to Liquibase (https://www.liquibase.org/).

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+#### Servidor
+* Host: `localhost`
+* Port: `5432`
+* Nome do banco: `central_register`
+#### Autenticação
+* Autenticação: `Database Native`
+* Usuário: `postgres`
+* Senha: `postgres`
+
+Com o banco criado, você só precisa rodar a aplicação, e ela criará as tabelas sozinha porque estamos usando a biblioteca do Liquibase. Feito isso, você pode acessar a nossa documentação como mostrado abaixo.
```
Comment (gabrieloswaldo) [inline README.md]: Same here.

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
...
+Before running our application, you have to create a postgres database locally with the following attributes:
+#### Server
+* Host: `localhost`
+* Port: `5432`
+* Database name: `central_register`
+#### Authentication
+* Authentication: `Database Native`
+* Username: `postgres`
+* Password: `postgres`
+
```
Comment (gabrieloswaldo) [inline README.md:18]: The Lombok dependency used in the project requires a plugin in the IDE (at least for IntelliJ). Maybe it's a good ideia to say it here.

Comment (izumizawa) [reply README.md:18]: wow I didn't know about this hahahah

File: README.md [hunk 5]
```diff
@@ -1 +1,51 @@
...
+Also, we used the [Lombok library](https://projectlombok.org/) in the project, so if you open it in an IDE, like IntelliJ, you should install a plugin to analyze it better.
```
Comment (gabrieloswaldo) [inline README.md]: I think here maybe it's better 'to run the application without errors.' intead of 'to analyze it better.'

**Review by gabrieloswaldo — APPROVED** [review]

**Review by rgueltamatera — APPROVED** [review]
this is looking really good :)

**Pull request merged**